qiskit/assembler/assemble_circuits.py

```diff
@@ -118,7 +124,7 @@ def _assemble_circuit(
 
     instructions = []
     for op_context in circuit.data:
-        instruction = op_context.operation.assemble()
+        instruction = op_context.operation._assemble()
```

Comment:
By calling the private method, the functionality is supported until the removal of `_assemble_circuit`